tests/config/nf-test.config

```diff
@@ -2,7 +2,7 @@ params {
     publish_dir_mode = "copy"
     singularity_pull_docker_container = false
     test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules'
-    modules_test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/3e79d4b5fbed8c6ef7546d50696b343ab9a95fbe/'
+    testdata_base_path = 's3://ngi-igenomes/testdata/nf-core/modules/'
```

Comment:
Actually if we want to use files from github, we can use
```
modules_test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/
```
and we're good, so we still are backward compatible-ish.

We could even update https://github.com/nf-core/modules/blob/master/tests/config/test_data.config so that data is in test_data_base and not in the paths, and we can easily switch from s3 to github

Reply:
nevermind actually, since we plan to drop the usage of this file, we actually don't need anything done

Reply:
just `modules_test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'` and we're good to go

Reply:
no, but it's still a possibility, that all I'm saying

Reply:
What we achieve here is tests portability, and we proved that by mirroring all we have on GitHub to s3.
Now one could test with s3, github links or local path

Reply:
Do we have a :heavy_check_mark: from @drpatelh?

Reply:
so I did test with local path, and it works